en/heroku/README.md

```diff
@@ -100,20 +108,13 @@ db_from_env = dj_database_url.config(conn_max_age=500)
 DATABASES['default'].update(db_from_env)
 ```
 
+Note: When you add `whitenoise.middleware.WhiteNoiseMiddleware` to the  `MIDDLEWARE` list make sure it is placed after the django `SecurityMiddlewear`, and leave in the other django middlewear.
```

Comment:
```suggestion
Note: When you add `whitenoise.middleware.WhiteNoiseMiddleware` to the  `MIDDLEWARE` list make sure it is placed after the Django `SecurityMiddleware`, and keep the other Django middleware.
```

Reply:
Does "middleware" have a plural? "middlewares"? If so, we might want to use it at the end of this sentence instead of the singular.

Reply:
Yes, middlewares is the plural, I will make the amendment as suggested ✨